src/main/scala/com/datastax/gatling/plugin/model/DseGraphAttributesBuilder.scala

```diff
@@ -158,23 +139,22 @@ case class DseGraphRequestAttributes(attr: DseAttributes) {
     * @param readCL Consistency Level to use
     * @return
     */
-  def withReadConsistency(readCL: ConsistencyLevel) = DseGraphRequestAttributes(attr.copy(graphReadCL = Some(readCL)))
+  def withReadConsistency(readCL: ConsistencyLevel) = DseGraphAttributesBuilder(attr.copy(readCL = Some(readCL)))
 
   /**
     * Define [[ConsistencyLevel]] to be used for write queries
     *
     * @param writeCL Consistency Level to use
     * @return
     */
-  def withWriteConsistency(writeCL: ConsistencyLevel) = DseGraphRequestAttributes(attr.copy(graphWriteCL = Some(writeCL)))
+  def withWriteConsistency(writeCL: ConsistencyLevel) = DseGraphAttributesBuilder(attr.copy(writeCL = Some(writeCL)))
 
 
   /**
     * Backwards compatibility to set consistencyLevel
     *
     * @deprecated
```

Comment:
If this is deprecated should we use the annotation too?

Reply:
+1